README.md

```diff
@@ -1,34 +1,63 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# SILACAK
+
+A progressive web app (PWA) to track COVID19 cases.
+
+## Purpose and Principles
+
+Help field officer (TNI) to easily tracing the cases. According to WHO Tracing ratio should be 1:15.
```

Comment:
```suggestion
The main purpose of Silacak is to help our field officers (TNI) easily trace COVID-19 cases in Indonesia. According to WHO Tracing ratio should be 1:15.
```

- Grammar.
- For the second sentence, we need to provide links to any official source to further drive the purpose of this project.